libsnowflakeclient/deps-build/darwin/curl/bin/curl-config

```diff
@@ -160,15 +172,15 @@ while test $# -gt 0; do
 
     --static-libs)
         if test "Xyes" != "Xno" ; then
-          echo ${exec_prefix}/lib/libcurl.a -L/Users/kristopherwagner/libsnowflakeclient/scripts/../deps-build/darwin/openssl/lib -lssl -lcrypto -lz -ldl
+          echo ${exec_prefix}/lib/libcurl.a -L/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/oob/lib -L/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/openssl/lib -lssl -lcrypto -lz -ltelemetry -ldl
         else
           echo "curl was built with static libraries disabled" >&2
           exit 1
         fi
         ;;
 
     --configure)
-        echo " '--with-ssl=/Users/kristopherwagner/libsnowflakeclient/scripts/../deps-build/darwin/openssl' '--without-nss' '--enable-static' '--disable-shared' '--prefix=/Users/kristopherwagner/libsnowflakeclient/scripts/../deps-build/darwin/curl' '--without-libssh2' '--disable-rtsp' '--disable-ldap' '--disable-ldaps' '--disable-telnet' '--disable-tftp' '--disable-imap' '--disable-smb' '--disable-smtp' '--disable-gopher' '--disable-pop3' '--disable-ftp' '--disable-dict' '--disable-file' '--disable-manual' 'CC=clang' 'CFLAGS=-arch x86_64 -Xarch_x86_64 -DSIZEOF_LONG_INT=8 -mmacosx-version-min=10.12' 'LIBS=-ldl'"
+        echo " '--with-ssl=/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/openssl' '--without-nss' '--enable-static' '--disable-shared' '--prefix=/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/curl' '--without-libssh2' '--disable-rtsp' '--disable-ldap' '--disable-ldaps' '--disable-telnet' '--disable-tftp' '--disable-imap' '--disable-smb' '--disable-smtp' '--disable-gopher' '--disable-pop3' '--disable-ftp' '--disable-dict' '--disable-file' '--disable-manual' 'CC=clang' 'CFLAGS=-arch x86_64 -Xarch_x86_64 -DSIZEOF_LONG_INT=8 -mmacosx-version-min=10.12' 'LDFLAGS=-L/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/oob/lib' 'LIBS=-ltelemetry -ldl' 'CPPFLAGS=-I/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/oob/include'"
```

Comment:
Paths include local fixed paths instead of relative paths to the base project directory.